etc/ci/performance/test_all.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+case "$1" in
```

Comment:
`set -o nounset` will make this an error in case no arguments are passed, but the usage indicates this should be allowed (IIUC defaulting to the `servo` engine).